src/Jackett.Common/Definitions/hdbao.yml

```diff
@@ -0,0 +1,297 @@
+
+login:
+  path: login.php
+  method: form
+  form: form[action="takelogin.php"]
+  inputs:
+    secret: ""
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    two_step_code: "{{ .Config.2facode }}"
+    logout: ""
+    securelogin: ""
+    ssl: yes
+    trackerssl: yes
+  error:
+    - selector: td.embedded:contains("登录失败！")
+  test:
+    path: index.php
+    selector: a[href="logout.php"]
+
+search:
+  paths:
+    - path: torrents.php
+  inputs:
+    $raw: "{{ range .Categories }}cat{{.}}=1&{{end}}"
+    search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    # 0: include dead, 1: alive, 2: dead
+    incldead: 0
+    spstate: "{{ if .Config.freeleech }}2{{ else }}0{{ end }}"
+    # 0: title, 1: description, 3: uploader, 4: IMDB URL
+    search_area: "0"
+    # 0: AND, 2: exact
+    search_mode: 0
+    sort: "{{ .Config.sort }}"
+    type: "{{ .Config.type }}"
+    # TODO: The example I copied, `./freefarm.yml`, includes a
+    # `notnewword` key but I don't see that key from submitting the
+    # search form in the web UI of either that example or this
+    # indexer. What am I missing?
+    #
+    # TODO: On that note, while inspecting the payloads of both search
+    # forms I noted a bunch of parameters that the web UI sends that
+    # aren't included in that example. How does that reconcile with
+    # "we don't want an obvious 'Jackett footprint'" from my last PR
+    # feedback especially given how much of Jackett/Prowlarr's traffic
+    # is submissions of this form?
+
+  rows:
+    selector: table.torrents > tbody > tr:has(a[href^="download.php?id="])
+
+  fields:
+    category:
+      selector: a[href^="?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title_default:
+      selector: a[href^="details.php?id="]
+    title:
+      selector: a[title][href^="details.php?id="]
+      attribute: title
+      optional: true
+      default: "{{ .Result.title_default }}"
+    details:
+      selector: a[href^="details.php?id="]
+      attribute: href
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    poster:
+      selector: img[data-src]
+      attribute: data-src
+    imdbid:
+      # site currently only has a badge and rating, the id is not present. just in case a future update.
+      selector: a[href*="imdb.com/title/tt"]
+      attribute: href
+    doubanid:
+      # site currently only has a badge and rating, the id is not present. just in case a future update.
+      selector: a[href*="movie.douban.com/subject/"]
+      attribute: href
```

Comment:
```suggestion
```

They don't seem to be using these at all.

Reply:
I assume you also meant to remove the `imdbid:` line above, too. That's what I've committed, so LMK if I should keep it and why if so.